foxglove_bridge_base/include/foxglove_bridge/websocket_server.hpp

```diff
@@ -194,6 +194,7 @@ class Server final : public ServerInterface {
   explicit Server(std::string name, LogCallback logger,
                   const std::vector<std::string>& capabilities,
                   const std::vector<std::string>& supportedEncodings = {},
+                  const std::unordered_map<std::string, std::string>& metadata = {},
```

Comment:
For simplicity, I decided to make this a map of string-string pairs. When e.g. using `std::variant`, we would have to provide json conversions for it.

Reply:
I made an indifferent comment on the spec PR, but after seeing this I think we should change the spec to `<string, string>`